orchestrator/src/main/java/com/scottlogic/deg/orchestrator/generate/GenerateCommandLine.java

```diff
@@ -151,6 +160,11 @@ public boolean overwriteOutputFiles() {
         return this.overwriteOutputFiles;
     }
 
+    @Override
+    public boolean useStdOut() {
+        return outputPath == STD_OUT;
```

Comment:
Typically this should be using `equals(...)` instead of `==`. While in this case they are equivalent, in terms of logic we care about the two objects having the same properties, not being the same instance.

Reply:
in this case we care about the same instance, not the properties.

Reply:
In that case the instance is serving a different purpose to being a Path.